### PR TITLE
Fix: Update Vsphere Integration Test

### PR DESCRIPTION
**Pull request by mohamed-rafraf**
**What this PR does / why we need it**:
This PR update the Vsphere Integration test config in order to fix the failures in `pre-kubermatic-test-integration` prow job

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by moadqassem — CHANGES_REQUESTED** [review]
Please change the datastore name from vSAN to  ```Datastore0-truenas```

File: cmd/conformance-tester/pkg/scenarios/vsphere.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (s *vSphereScenario) Cluster(secrets types.Secrets) *kubermaticv1.ClusterSp
 	}
 
 	if s.datastoreCluster {
-		spec.Cloud.VSphere.DatastoreCluster = "dsc-1"
+		spec.Cloud.VSphere.DatastoreCluster = "Datastore0-truenas"
```
Comment (moadqassem) [inline cmd/conformance-tester/pkg/scenarios/vsphere.go:84]: I am not sure if this is gonna work as this is not a datastore cluster. Datastore cluster is a collection of ESXi hosts that use the local storage of each of those hosts(VMFS)

**Comment by xrstf** [comment]
/approve
/lgtm

Thanks a bunch :)

File: cmd/conformance-tester/pkg/scenarios/generator.go [hunk 2]
```diff
@@ -184,7 +184,7 @@ func providerScenario(
 		scenario := &vSphereScenario{baseScenario: base}
 		scenario.customFolder = opts.ScenarioOptions.Has("custom-folder")
 		scenario.basePath = opts.ScenarioOptions.Has("basepath")
-		//scenario.datastoreCluster = opts.ScenarioOptions.Has("datastore-cluster")
+		// scenario.datastoreCluster = opts.ScenarioOptions.Has("datastore-cluster")
```
Comment (moadqassem) [inline cmd/conformance-tester/pkg/scenarios/generator.go]: You need tot disable the prow job https://github.com/kubermatic/kubermatic/blob/main/.prow/provider-vsphere.yaml#L153

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**